price_pusher/README.md

```diff
@@ -71,6 +71,14 @@ npm run start -- injective --grpc-endpoint https://grpc-endpoint.com \
     [--pushing-frequency 10] \
     [--polling-frequency 5] \
 
+# For Aptos
```

Comment:
Nitpick :
Update `endpoint`,  `price-service-endpoint`, `pyth-contract-address`  to match `config.aptos.testnet.sample.json`